app/layout.tsx

```diff
@@ -0,0 +1,26 @@
+import CommonLayout from "@/common/CommonLayout";
+import type { Metadata } from "next";
+import { Inter } from "next/font/google";
+import React from "react";
+import "../style/global.css";
+
+const inter = Inter({ subsets: ["latin"] });
+
+export const metadata: Metadata = {
+  title: "Follow Detector",
+  description: "Following by Follow Detector App",
+};
+
+export default function RootLayout({
+  children,
+}: Readonly<{
+  children: React.ReactNode;
+}>) {
+  return (
+    <html lang="ko">
+      <CommonLayout>
+        <body className={inter.className}>{children}</body>
+      </CommonLayout>
```

Comment:
CommonLayout이 body 태그로 이루어져 있는데, 하위에 body 태그가 또 있어서 중복 코드 같습니다!
폰트 provide 해주는 body 태그까지 한번에 CommonLayout에 넣어주고 여기 body 태그는 삭제하면 더 좋을것 같습니당💪🏻
```suggestion
      <CommonLayout>
       {children}
      </CommonLayout>
```